dso-l2/src/test/java/com/tc/net/groups/TCGroupManagerImplTest.java

```diff
@@ -561,13 +561,13 @@ protected String getVersion() {
 
     // Incompatible version, higher weights. Close down the handshake, the other guy will zap himself.
     MessageChannel channel1 = mockMessageChannel();
-    tcGroupManager.receivedHandshake(mockHandshakeMessage(channel1, "4.3.1", new long [] { 4, 5 }));
+    tcGroupManager.receivedHandshake(mockHandshakeMessage(channel1, "5.3.1", new long [] { 4, 5 }));
     verify(channel1).close();
```

Comment:
This implies that VERSION_COMPATIBILITY_CHECK property is set to true. Setting it explicitly in the code would require initialising the static class TCPropertiesImpl by PowerMock (see ClientHandshakeManagerTest above), and then setting other properties explicitly as well, so decided to leave the implicit configuration as is. Changed the version difference from minor to major to get the desired failure.

Reply:
In order to set the default for the platform change this line to false

https://github.com/Terracotta-OSS/terracotta-core/blob/master/common/src/main/resources/com/tc/properties/tc.properties#L330

Reply:
Not sure we want to change the default yet? I think myself and Chris D agreed just to ensure that it is configurable.